src/main/webapp/WEB-INF/view/navbar.jsp

```diff
@@ -0,0 +1,12 @@
+<nav>
+  <a id="navTitle" href="/">CodeU Chat App - Nemo</a>
+  <a href="/conversations">Conversations</a>
+  <% if(request.getSession().getAttribute("user") != null){ %>
```

Comment:
is it possible to use a variable so we don't have to keep calling these methods?

Reply:
I'll update in the next PR.